tests/sqlite3/test-sqlite3.cpp

```diff
@@ -641,6 +641,21 @@ TEST_CASE("SQLite DDL with metadata", "[sqlite][ddl]")
     CHECK(ddl_t3_found == false);
 }
 
+TEST_CASE("SQLite DDL symmetry", "[sqlite][ddl][symmetry]")
```

Comment:
"symmetry" should really be replaced with "round trip"

Reply:
Will replace everything with Roundtrip